### ISSUE TITLE
cloth equipment is lost too easily to flame

### ISSUE THREAD

**Issue by rmsr** [sneezymud/sneezymud#88]
A person wrote on Facebook, "...How quickly fire destroys even high level equipment. Even low level flamestrike can nuke 2-4 pieces of sylvan in one try and it is destroyed before you can even react to leave the battle. I see three issues, here: How easily things are affected by fire (especially considering level difference), how quickly items are consumed (regardless of quality) and the difficulty of extinguishing, both what you are where and things in your inventory."

**Comment by rmsr** [comment]
I think flammable inventory is just the penalty you pay for not keeping your stuff in a closed metal fishing pail haha. The worn equipment side of things, though, maybe make charred gear fixable by high-level player practitioners? How does the protection from fire spell affect the flammability of worn gear?

**Comment by Aion2501** [comment]
Yeah, thanks to Trawl this was added to tweakables and is currently set to a negligible chance of occuring. However, when it does, POOF! Or shatter. There are no mitigating factors. IF it happens, your day just got worse. 

I did already write up an issue on this weeks ago, hoping we could figure out a way for cold/fire resist to mitigate the damage and add a new resistance feature to Advanced Adventuring. It just hasn't happened yet with other things taking priority.

**Comment by Leyline77** [comment]
Yeah but high level silk / magical could be considered as a different material then common level silk / common level wood.  I mean  it's Elfish Sylvan Armor, that's not "just silk"
